src/features/ballot/components/BallotOverview.tsx

```diff
@@ -38,9 +43,14 @@ function BallotOverview() {
   const sum = sumBallot(ballot?.votes);
 
   const allocations = ballot?.votes ?? [];
-  const canSubmit = router.route === "/ballot" && allocations.length;
+  const isBallotPage = router.route === "/ballot";
+  const canSubmit = isBallotPage && allocations.length;
   const isProjectsPage = router.route === "/projects";
 
+  useEffect(() => {
+    //
+  }, [ballot]);
```

Comment:
is this intentional?

Reply:
While testing this made it more responsive